### PR TITLE
Dashboard and Editor 

### PR DESCRIPTION
**Pull request by humzahasan**
PR Description " Dashboard and Editor page added

[Please check me only](https://github.com/humzahasan/notova/pull/9/files/13f9ece9b530003e688daa5c363f866365fd5cab..ec892fd936c21374632fb20cc4a3ee913efb561a)

### CODE REVIEW

File: src/pages/Editor/Editor.css [hunk 1]
```diff
@@ -31,4 +31,54 @@
...
+  align-items: center;
+  margin: 1rem;
+}
+
+.select {
+  border: 1px solid var(--primary-color);
+  border-radius: 5px;
+}
+
+/*
```
Comment (humzahasan) [inline src/pages/Editor/Editor.css:52]: Will be removing this in next PR! Ignore here!

File: src/pages/Editor/Editor.jsx [hunk 2]
```diff
@@ -1,29 +1,106 @@
...
+            <option value="Teams" />
+            <option value="School" />
+            <option value="College" />
+            <option value="Exercise" />
+            <option value="Personal" />
+            <option value="Creativity" />
+            <option value="Other" />
+          </datalist>
+
+          {/* <input
```
Comment (humzahasan) [inline src/pages/Editor/Editor.jsx:82]: Will be removing this in next PR, ignore here!

File: src/context/note-context.js [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+      const res = await axios.get("/api/notes", {
+        headers: {
+          authorization: getToken(),
+        },
+      });
+      if (res.status === 200) {
+        dispatch({ type: "SET_NOTES", payload: res.data.notes });
+      }
+    } catch (error) {
+      console.log(error.response);
```
Comment (iRohitGaur) [inline src/context/note-context.js:30]: suggestion: you can use `toast` here as well

File: src/pages/Editor/Editor.jsx [hunk 4]
```diff
@@ -37,13 +114,31 @@ const Editor = () => {
...
+              Add My Note
+            </button>
+          )
+        ) : (
+          <button disabled="disabled" className="btn btn-outline btn-disabled">
+            Add My Note
+          </button>
+        )}
       </div>
+      {/* <ToastContainer /> */}
```
Comment (iRohitGaur) [inline src/pages/Editor/Editor.jsx:141]: suggestion: remove commented code

File: src/pages/Login/Login.jsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+      const res = await axios.post("/api/auth/login", { email, password });
+      console.log(res);
+      if (res.status === 200 && res.data.encodedToken) {
+        setToken(res.data);
+        setUser(res.data.foundUser);
+        navigate("/mynotes");
+      }
+    } catch (error) {
+      console.log(error);
+      console.log(error.response);
```
Comment (iRohitGaur) [inline src/pages/Login/Login.jsx:23]: suggestion: remove `console.log`. use toast

File: src/pages/MyNotes/MyNotes.jsx [hunk 6]
```diff
@@ -1,8 +1,64 @@
...
+          notes.filter((note) => note[filterType]?.includes(filterText))
+        )
+      : setMyNotes(notes);
+  }, [filterType, filterText, notes]);
+
+  useEffect(() => {
+    if (sortType.length > 0) {
+      if (sortType === "date-oldest") {
+        setMyNotes(
+          notes?.sort((a, b) => {
```
Comment (iRohitGaur) [inline src/pages/MyNotes/MyNotes.jsx:33]: suggestion: `sort` is not a pure function. Make this a pure update.

File: src/pages/MyNotes/MyNotes.jsx [hunk 7]
```diff
@@ -11,7 +67,88 @@ const MyNotes = () => {
...
+                        date: new Date(),
+                        title: title,
+                        content: content,
+                        tag: tag,
+                        priority: priority,
+                        color: color,
+                      });
+                    }}
+                    colorHandler={() => {
+                      console.log("color", cardColor);
```
Comment (iRohitGaur) [inline src/pages/MyNotes/MyNotes.jsx:141]: suggestion: remove `console.log`

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Gave some suggestions. Please look into it.

Good to merge 👍🏻

File: src/Components/Card/Card.jsx [hunk 8]
```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import "./Card.css";
+import editIcon from "../../assets/edit.svg";
+import unarchiveIcon from "../../assets/undo.svg";
+import archiveIcon from "../../assets/archive.svg";
+import deleteIcon from "../../assets/delete.svg";
+import colorIcon from "../../assets/color.svg";
+import permDeleteIcon from "../../assets/permanent.svg";
```
Comment (DivyanshSareen) [inline src/Components/Card/Card.jsx:8]: Suggestion: can use index.js here

**Review by DivyanshSareen — APPROVED** [review]
The code is easy to read and navigate. Good naming convention on the handler functions. Good to merge

**Pull request merged**